client/src/components/Projects/index.js

```diff
@@ -1,16 +1,49 @@
 import React, { Component } from 'react';
 import './styles.css';
+import ProjectService from '../../services/projects';
 
 /** Projects page component */
 class Projects extends Component {
+/**
+	 * A Projects component instance
+	 * @constructor
+	 * @param {*} props - The properties passed into this component
+	 */
+	constructor(props) {
+		super(props);
+		this.projectService = new ProjectService();
+		this.state = {
+			projects: [],
+			newProject: {}
+		};
+		this.getProjects();
+	}
+    
 	render() {
 		return (
 			<div className="container">
 				<h2>Projects</h2>
 				<div className="title-underline bg-theme"></div>
+				<div>
+					<div>
+						<h5>Monday, March 13</h5>
```

Comment:
Alright, so the projects will be displayed as an entirety, not by day like the tasks